test/jsupdate-behaviour-mocha.js

```diff
@@ -61,14 +136,40 @@ describe("framework-state", function() {
                 network.graph.addEdge('sink', 'output', 'sut', 'input');
                 test.sendData(network.processes.sink.component, 'input', "tongue");
                 setTimeout(done.bind(this, "nothing happened"), 100);
-            });
+            }).catch(fail);
+        }).should.become("nothing happened");
+    });
+    it("should not fire updater if upstream updater did not produce an output state after producing one", function() {
```

Comment:
s/did not produce an output state after producing one/returns undefined after producing an output state/